tests/sparseml/export/transformers/test_transformers.py

```diff
@@ -102,13 +111,15 @@ def test_export_multiple_files(self, setup):
             single_graph_file=False,
         )
 
-    @pytest.mark.skipif(
-        reason="skipping since this functionality needs some more attention"
-    )
-    def test_export_validate_correctness(self, setup):
+    def test_export_validate_correctness(self, caplog, setup):
+        if self.is_model_quantized:
+            pytest.skip(
+                "Skipping since quantized models may not pass this test"
+                "due to differences in rounding between quant ops in PyTorch and ONNX"
+            )
```

Comment:
Is there an expected error range here that we could check for rather than skipping entirely?